### PR TITLE
Add min max properties to Input and pkg-pr-new workflow

### PR DESCRIPTION
**Pull request by Michele-Masciave**
Add minLength maxLength properties to Input. Also, added pkg-pr-new.yml.

### CODE REVIEW

**Review by neotob — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Added the pkg.pr.new workflow
```
Comment (neotob) [inline CHANGELOG.md:12]: not 100% sure if we normally added this to the changelog @manni497 ?

Comment (neotob) [reply CHANGELOG.md:12]: yes we did sorry 😄

**Review by manni497 — APPROVED** [review]

File: src/lib/Input.tsx [hunk 2]
```diff
@@ -56,6 +72,9 @@ const Input = <T extends FieldValues>(props: InputProps<T>) => {
   if (step && type !== "number" && type !== "range") {
     throw new Error("step can only be used with number or range inputs");
   }
+  if ((minLength || maxLength) && type && !["text", "password", "textarea", "search", "tel", "url", "email"].includes(type)) {
```
Comment (manni497) [inline src/lib/Input.tsx:75]: @Michele-Masciave Passing 0 will throw an exception :) 

I would check also that values are > 0 :)

**Review by manni497 — CHANGES_REQUESTED** [review]

File: src/lib/Input.tsx (hunk 2 above)
Comment (Michele-Masciave) [reply src/lib/Input.tsx]: Done as suggested

**Review by manni497 — APPROVED** [review]

**Pull request merged**